jest.config.js

```diff
@@ -30,5 +30,7 @@ module.exports = {
   coverageDirectory: 'coverage',
   coverageProvider: 'v8',
   testPathIgnorePatterns: ['dist'],
-  modulePathIgnorePatterns: ['/e2e/']
+  modulePathIgnorePatterns: ['/e2e/'],
+  globalSetup: './utils/jest-global-setup.ts',
```

Comment:
nit: maybe we should move all these inside `__tests__`, WDYT?  Including server.ts and test related files.

Reply:
If we move them to `__test__` we would need to ignore them similar to `/e2e/` tests. I don't have a strong preference here.

Reply:
I've moved them to `__test__`